usagov/usagov-2021
# USAGOV-1107-FAQ-Quotes

Use double quotes around answers and escape quotes inside of answers


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1107

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



- Visit a page like /hawaii-wildfires
- Edit the page to select the checkbox "FAQ Page"
- Save and view the page
- Inspect the code and search for "FAQPage"
- Verify that the "text" items:
  - have double quotes around the value
  - have a forward slash before any quotes within the value

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
